modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/OperationMessageProcessorBuilder.java

```diff
@@ -52,7 +54,10 @@ protected OperationMessageProcessor createMessageProcessor(ExtensionManager exte
 
     final boolean supportsOAuth = supportsOAuth(extensionModel);
     if (operationModel.getModelProperty(PagedOperationModelProperty.class).isPresent()) {
-      resultTransformer = getPagingResultTransformer(operationModel, extensionConnectionSupplier, supportsOAuth).orElse(null);
+      resultTransformer = getPagingResultTransformer(operationModel, extensionConnectionSupplier, supportsOAuth,
+                                                     initialSpanInfoProvider
+                                                         .getInitialSpanInfo(NULL_COMPONENT, OPERATION_EXECUTION_SPAN_NAME, ""))
```

Comment:
Same concern here. Isn't this span info going to be used for the `getConnection`? Shouldn't the name be `GET_CONNECTION_SPAN_NAME`?
And I'm not entirely sure if the component needs to be `NULL_COMPONENT` in this context. We are in the process of building a `Component`, so it looks we are close to actually being able to use the real one.
Then again, I'm not sure about the implications.

Reply:
Still looking into it.

Reply:
I've set the component preventing it from being generated each time.